src/Xamarin.Android.Build.Tasks/Tasks/BuildApk.cs

```diff
@@ -146,22 +146,28 @@ void ExecuteWithAbi (string [] supportedAbis, string apkInputPath, string apkOut
 					var lastWriteInput = File.GetLastWriteTimeUtc (apkInputPath);
 					using (var packaged = new ZipArchiveEx (apkInputPath, FileMode.Open)) {
 						foreach (var entry in packaged.Archive) {
-							Log.LogDebugMessage ($"Deregistering item {entry.FullName}");
-							existingEntries.Remove (entry.FullName);
+							// NOTE: aapt2 is creating zip entries on Windows such as `assets\subfolder/asset2.txt`
+							var entryName = entry.FullName;
+							if (entryName.Contains ("\\")) {
```

Comment:
Very possibly a silly question, but on the small chance it might be useful make this code match f356220827c55a488ccf5732e6fd7e4e3333f55a even more closely, this could pass a `char` argument:

```cs
if (entryName.Contains ('\\')) {
```

So it would match:
https://github.com/xamarin/xamarin-android/blob/61fd95e52f2466888c7f6c6a2d0158599fef4723/src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveEx.cs#L120

Reply:
Actually, I think the `"\\"` one should be used. The `ZipArchiveEx` is using:
```csharp
public static class Enumerable
{
    public static bool Contains<TSource>(this IEnumerable<TSource> source, TSource value);
````
I've seen this one is a bit slower than `string.Contains` ^^

Reply:
Ah, interesting. Sounds good!